README.md

```diff
@@ -31,7 +31,7 @@ shapeless is part of the [Typelevel][typelevel] family of projects. It is an Ope
 License v2, hosted on [github][source]. Binary artefacts are published to the
 [Sonatype OSS Repository Hosting service][sonatype] and synced to Maven Central.
 
-Most discussion of shapeless and generic programming in Scala happens on the shapeless [Discord][discord]. There
+Most discussion of shapeless and generic programming in Scala happens on the [Typelevel Discord][discord]. There
```

Comment:
Maybe "on the #shapeless channel of the Typelevel Discord" ...

Reply:
Updated!